src/Compilers/CSharp/Portable/Binder/Binder_Lookup.cs

```diff
@@ -763,6 +877,25 @@ internal virtual bool SupportsExtensionMethods
         {
         }
 
+#nullable enable
+        /// <summary>
+        /// Indicates whether a scope should be inspected for candidate extension types
+        /// when looking up the scope chain.
+        /// </summary>
+        internal virtual bool SupportsExtensionTypes => false;
```

Comment:
>SupportsExtensionTypes [](http://example.com/codeflow?start=30&length=22)

I think we should simply reuse ```SupportsExtensionMethods```. Possibly renaming it to indicate that it covers more scenarios.